infra/app/app-config/env-config/outputs.tf

```
@@ -7,3 +7,9 @@ output "database_config" {
     schema_name        = var.app_name
   } : null
 }
+
+output "incident_management_service_integration" {
+  value = var.has_incident_management_service_integration ? {
+    ssm_incident_management_secret_name = "/monitoring/${var.app_name}/${var.environment}/incident-management-integration-url"
```

nit: looks good, here are some quick ideas for making the variable names just a bit shorter while still retaining the key information: 

also calling it `param name` instead of `secret name` since we're using "AWS Parameter Store" not "Secret Manager"

```suggestion
output "incident_management_service" {
  value = var.has_incident_management_service_integration ? {
    integration_url_param_name = "/monitoring/${var.app_name}/${var.environment}/incident-management-integration-url"
```

---

done